tutorials/speech2srt/speech2srt.py

```diff
@@ -139,13 +127,23 @@ def main():
     parser.add_argument(
         "--out_file",
         type=str,
-        default="en",
+        default="subtitle",
     )
     parser.add_argument(
         "--max_chars",
         type=int,
         default=40,
     )
+    parser.add_argument(
+        "--encoding",
+        type=str,
+        default='LINEAR16'
+    )
+    parser.add_argument(
+        "--audio_channel_count",
+        type=int,
+        default=1
+    )
```

Comment:
@lepistom Added these parameters to support other file types and multi-channel audio files.